### ISSUE TITLE
Worker panicked at 'called `Result::unwrap()`

### ISSUE THREAD

**Issue by sammy007** [ethcore/parity#2063]
```
thread 'IO Worker #3' panicked at 'called `Result::unwrap()` on an `Err` value: Utf8Error { valid_up_to: 40 }', ../src/libcore/result.rs:785
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

Linux, parity 1.3

**Comment by keorn** [comment]
Can you provide more context? What were you trying to do? Is it possible to reproduce with `RUST_BACKTRACE=1`?

**Comment by sammy007** [comment]
It's mining node. Since this node is production, I can't run it with backtrace.

**Comment by arkpar** [comment]
any non-ascii characters in the database path?

**Comment by sammy007** [comment]
No, path is default `/home/ascii/.parity`. I noticed that 2 times today during 2 months of normal flight.

```
2016-09-09 14:07:31 Imported #2227432 ad44…95e0 (3 txs, 0.08 Mgas, 0.81 ms, 0.91 KiB)
thread 'IO Worker #0' panicked at 'called `Result::unwrap()` on an `Err` value: Utf8Error { valid_up_to: 40 }', ../src/libcore/result.rs:785
note: Run with `RUST_BACKTRACE=1` for a backtrace.
2016-09-09 14:08:01     0/95/96 peers     71 MiB db  364 MiB chain    2 KiB queue   64 KiB sync
2016-09-09 14:08:22 Imported #2227433 ed84…a500 (23 txs, 0.57 Mgas, 5.05 ms, 3.41 KiB)
--
2016-09-09 14:09:52 Imported #2227441 def3…2a5d (6 txs, 0.19 Mgas, 1.78 ms, 1.42 KiB)
thread 'IO Worker #3' panicked at 'called `Result::unwrap()` on an `Err` value: Utf8Error { valid_up_to: 40 }', ../src/libcore/result.rs:785
note: Run with `RUST_BACKTRACE=1` for a backtrace.
2016-09-09 14:10:00     0/18/96 peers     71 MiB db    2 MiB chain    2 KiB queue   10 KiB sync
2016-09-09 14:10:08 Imported #2227442 de69…2a51 (1 txs, 0.04 Mgas, 0.67 ms, 0.69 KiB)
```

**Comment by arkpar** [comment]
Probably fixed with #2217

**Comment by arkpar** [comment]
Please reopen if it appears again.
